modules/thermal_hydraulics/src/userobjects/SolidMaterialProperties.C

```diff
@@ -73,3 +73,21 @@ SolidMaterialProperties::rho(Real temp) const
 {
   return _rho.value(temp, Point());
```

Comment:
Are these still needed? I'd think the AD ones would replace them.

Reply:
I think you are right. I was not sure if RELAP-7 would be using this - but most likely not. I will try to remove it and let's see...